frontend/public/locales/en/common.json

```diff
@@ -246,6 +246,8 @@
   "more-type": "More {{type}}",
   "verified": "Verified",
   "app-is-verified": "This app is verified",
+  "verification-instructions": "Verify your app ID to indicate that your Flathub package is approved by the upstream developer. You'll need to confirm your identity, for example with a website or an account on a source code hosting site. The verified badge will be shown alongside the app name, publisher name, and the identity you use for verification.",
+  "verification-warning": "We may remove the verification badge from an app if it could mislead people about the app's official status.",
```

Comment:
We might want to keep this more general, just to be able to act on certain things? 

@ramcq for wording guidance

Reply:
The Flathub team may remove the verification badge from any app at our sole discretion, for example if it could mislead people about the app's official status or cause confusion regarding a trademark.